parallel-consumer-core/src/main/java/io/confluent/parallelconsumer/internal/AbstractParallelEoSStreamProcessor.java

```diff
@@ -1430,6 +1422,11 @@ private void clearCommitCommand() {
         }
     }
 
+    private boolean isTransactionCommittingInProgress() {
+        return options.isUsingTransactionCommitMode() &&
```

Comment:
it should be, but i am just playing it safe - IMHO - i prefer to explicitly check producer transaction state only if transactional producer is actually specified in options rather than check its state regardless and rely on default return to be false for non-transactional producer.